### PR TITLE
PMC Medical Contractor DLC

### PR DESCRIPTION
**Pull request by SELFHELL**
## About The Pull Request

Adds a medic to the PMC ERT, Loadouts consists of:

Advanced Medical/Defib Gloves
VP-70 Pistol with 2 Magazines
PMC SMG with 3 additional magazines
Medical HUD 
Pressurized BKTT Canister (Pouch)
Standard Medic Autoinjector Pouch
Full Lifesaver Rig
Lightpack Backpack

## Why It's Good For The Game

aleks

Should spice up the PMC ERT with a medic role that has been abnormally absent. Ill try and add some sprites if the whole concept gets the thumbs up from maints.
## Changelog
:cl:
add: PMC Medic
/:cl:

### CODE REVIEW

File: code/datums/jobs/job/pmc.dm [hunk 1]
```diff
@@ -68,6 +68,54 @@
...
 
+//PMC medic
+/datum/job/pmc/medic
+	title = "PMC Medic"
+	paygrade = "PMC2"
+	skills_type = /datum/skills/combat_medic/pmc
+	outfit = /datum/outfit/job/pmc/medic
+
+//medic loadout
+/datum/outfit/job/pmc/gunner
```
Comment (Xander3359) [inline code/datums/jobs/job/pmc.dm:79]:
```suggestion
/datum/outfit/job/pmc/medic
```

File: code/datums/jobs/job/pmc.dm [hunk 2]
```diff
@@ -68,6 +68,54 @@
...
+	name = "PMC Medic"
+	jobtype = /datum/job/pmc/medic
+
+	id = /obj/item/card/id/silver
+	belt = /obj/item/storage/belt/lifesaver/full
+	ears = /obj/item/radio/headset/distress/pmc
+	w_uniform = /obj/item/clothing/under/marine/veteran/pmc/webbing
```
Comment (Xander3359) [inline code/datums/jobs/job/pmc.dm:86]: This is a black webbing, so idk how you intend on putting a gun in a storage that can't hold it...

**Comment by Lumipharon** [comment]
They don't have medics because they're already exceptionally powerful.

**Review by Xander3359 — CHANGES_REQUESTED** [review]

File: code/datums/jobs/job/pmc.dm [hunk 3]
```diff
@@ -68,6 +68,54 @@
...
+	shoes = /obj/item/clothing/shoes/veteran/pmc
+	wear_suit = /obj/item/clothing/suit/storage/marine/veteran/pmc
+	gloves = /obj/item/defibrillator/gloves
+	head = /obj/item/clothing/head/helmet/marine/veteran/pmc/
+	mask = /obj/item/clothing/mask/gas/pmc
+	glasses = /obj/item/clothing/glasses/hud/health
+	suit_store = /obj/item/weapon/gun/smg/m25/elite/pmc
+	r_store = /obj/item/storage/pouch/medical_injectors/medic
+	l_store =/obj/item/storage/pouch/pressurized_reagent_pouch/bktt
+	back = /obj/item/storage/backpack/lightpack
```
[the remark this answers is not in the document]
Comment (SELFHELL) [reply code/datums/jobs/job/pmc.dm:96]: Didnt assume that they would ever need to recharge the glove defibs,

**Comment by Strayed-Raven** [comment]
Do not think PMC needs medics really. maybe a hard limit of 1

**Comment by Lumipharon** [comment]
Again, I still don't see WHY the strongest ERT by a significant margain needs this.

**Comment by Oscari-pog** [comment]
Because operator medics are badass and also really think that people exaggerate the consequence of a singular PMC medic appearing with the rest of the squad

He really isnt gonna be any game changing

**Comment by ivanmixo** [comment]
I agree, this isn't necessary. Sorry for taking so long

**Pull request closed without merging**